from app.exceptions import BusinessException


class OrderAlreadyExists(BusinessException):
    pass
